sources/packages/web/src/views/institution/locations/request-a-change/RequestAChangeFormSubmit.vue

```diff
@@ -139,7 +139,9 @@ export default defineComponent({
       activeOfferingId.value = application.offeringId;
       programs.value =
         await EducationProgramService.shared.getProgramsListForInstitutions();
-      selectedProgram.value = application.programId;
+      if (application.programIsActive) {
```

Comment:
For  the Mass offering change, I see the request change button is enabled for an offering of deactivated program.  
![image](https://github.com/bcgov/SIMS/assets/54600590/329a9930-1ec4-46d9-ab40-1a33986febe4)

Reply:
I am considering this as something for upcoming PR.